lib/osf-components/addon/components/file-browser/file-rename-modal/component.ts

```diff
@@ -32,7 +32,12 @@ export default class FileRenameModal extends Component<Args> {
 
     @action
     resetFileNameValue() {
-        this.newFileName =  this.originalFileName;
+        return this.newFileName =  this.originalFileName;
```

Comment:
Returning a value for a closure action is not necessary, as nothing is using the value here.

Reply:
Hey, my understanding is that not explicitly returning can be bad form. Not returning will not allow for potential optimization of the function since it will result in an undefined return value. Reference states it's not necessary, but undefined returns would decrease error handling abilities as well since the value would be implicitly returned as undefined. To implement this, I can add a try/catch block here in case an error occurs since naming collisions seem to also be causing bugs on the Front End during the POST request.

Reply:
Return value would be implicitly returned as undefined then. I don't know if this is best practice. I would like to implement it for the error handling we discussed during scrum but if returning explicitly would be detrimental we could remove it until the error PR.

Reply:
If whatever error handling to be done is dependent on the return value of an `action`, then we're probably looking at the wrong solution here. If we are looking to do validation on an input value, the validation should be done, either directly as a getter dependent on the value of `this.newName`, or as a validation object applied to a `changeset` based on the value of `this.newName`.
I suggest making this PR simple to solve the one problem identified, which is the bug where `this.originalFileName` is not set to the new file name after successful renaming.

Reply:
Ok, and those sound like good options to look into

Reply:
Let me know if this is good to resolve, otherwise we can touch base later to discuss